### DISCUSSION TITLE
手机端能否支持添加多个账号？

### DISCUSSION THREAD

**Discussion by dingdangdog** [community/community#38082] [Other Feature Feedback, Questions, & Ideas] [answered]
可添加多个账号，支持切换。

能否实现？或是否有计划实现？

**Comment by ossanna16** [comment, marked as answer]
Thanks for participating!
We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. What this means is that, until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
